### PR TITLE
[UI] Update MultiSend GUI to allow address labels

### PR DESCRIPTION
**Pull request by blondfrogs**
#446 
Added a Label section to the MultiSend GUI
MultiSend GUI will now show labels if address is selected from addressbook
MultiSend GUI now allows users to add or change the label to the address they are creating a multisend for
View MultiSend Button now displays the label in the print out along with the address and percentage

### CODE REVIEW

**Comment by blondfrogs** [comment]
@Mrs-X This is my first time really adding something to the GUI. Let me know what you think.

**Comment by Fuzzbawls** [comment]
@blondfrogs please provide before & after screenshots when submitting a PR that changes GUI layout/behavior

**Comment by blondfrogs** [comment]
Old GUI:
oldmultisend

**Comment by blondfrogs** [comment]
New GUI: 
multisendwithlabels

**Comment by blondfrogs** [comment]
@Fuzzbawls Screenshots showing the changes added

**Comment by presstab** [comment]
Looks good. Make sure that in the future, you checkout the branch under your own fork of pivx instead of under the main pivx repository. And then submit the PR from blondfrogs/pivx.

File: src/qt/multisenddialog.cpp [hunk 1]
```diff
@@ -71,6 +79,14 @@ void MultiSendDialog::on_viewButton_clicked()
 
     for (int i = 0; i < (int)pwalletMain->vMultiSend.size(); i++) {
         pMultiSend = pwalletMain->vMultiSend[i];
+        if (model && model->getAddressTableModel()) {
+            std::string associatedLabel;
+            associatedLabel = model->getAddressTableModel()->labelForAddress(pMultiSend.first.c_str()).toStdString();
+            if (associatedLabel != "(no label)") {
```
Comment (Warrows) [inline src/qt/multisenddialog.cpp]: Is this gonna fly after translation? Maybe use a bool to know if it is set instead of comparing a string.

Comment (blondfrogs) [reply src/qt/multisenddialog.cpp]: That is a good point. I am changing it print the label even if there is no label assigned. This way the users will see if correct label in the correct language.

**Comment by Sieres** [comment]
ACK

**Pull request merged**